{
    "checksum": "8557e8fe53aeaf1494cf8b4f8976901c2be0945061a54271579c5463efebd17a",
    "file": "exception.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 1,
    "package": "aravis",
    "path": "aravis/0.8.6-1/tests/exception.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/aravis/0.8.6-1/",
        "metric": {
            "size": 3424
        },
        "pts_link": "https://tracker.debian.org/pkg/aravis",
        "sloc": [
            [
                "ansic",
                30830
            ],
            [
                "xml",
                1528
            ],
            [
                "sh",
                274
            ],
            [
                "python",
                217
            ],
            [
                "makefile",
                60
            ],
            [
                "javascript",
                53
            ],
            [
                "cpp",
                6
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian-astro-team/aravis",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/aravis/0.8.6-1/tests/exception.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 1416,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.8.6-1"
}